06/VectorAddingElementsExample.cpp

```diff
@@ -0,0 +1,46 @@
+#include <iostream>
+#include <vector>
+#include "VectorAddingElementsExample.h"
+
+using namespace std;
+
+namespace samples
+{
+	void VectorAddingElementsExample()
+	{
+		vector<int> scoreList;
+		scoreList.reserve(7);
+
+		scoreList.push_back(30);
+		scoreList.push_back(50);
+		scoreList.push_back(80);
+		scoreList.push_back(65);
+		scoreList.push_back(73);
+
+		PrintScoreList(scoreList);
+
+		scoreList.pop_back();
+		scoreList.pop_back();
+
+		PrintScoreList(scoreList);
+
+		scoreList.resize(10);
+
+		PrintScoreList(scoreList);
+	}
+
+	void PrintScoreList(vector<int> scoreList)
+	{
+		vector<int>::iterator iter;
```

Comment:
put this one inside for loop.... vector<int>::iterator iter  = scoreList.begin()